AzureCalling/Controllers/LobbyViewController.swift

```diff
@@ -115,6 +125,15 @@ class LobbyViewController: UIViewController, UITextFieldDelegate {
         previewCenterImageView.isHidden = true
     }
 
+    private func openAudioDeviceDrawer() {
+        let audioDeviceSelectionViewController = storyboard?.instantiateViewController(withIdentifier: "AudioDeviceSelectionViewController") as! AudioDeviceSelectionViewController
+        audioDeviceSelectionViewController.audioDeviceSelectionManager = audioDeviceSelectionManager
+        audioDeviceSelectionViewController.audioDeviceTableDataSource = audioDeviceTableDataSource
+        audioDeviceSelectionViewController.modalPresentationStyle = .overCurrentContext
+        audioDeviceSelectionViewController.modalTransitionStyle = .crossDissolve
```

Comment:
Could `modalTransitionStyle` and `modalPresentationStyle` be defined in `audioDeviceSelectionViewController`?
Which could avoid duplicated call

Reply:
modalPresentationStyle must be defined before presenting, but the modalTransitionStyle can be done after.